### PR TITLE
ZDI assigns the following CVEs:

### PR DESCRIPTION
**Pull request by zdi-team**
M 2020/27xxx/CVE-2020-27858.json
 M 2020/27xxx/CVE-2020-27859.json

### CODE REVIEW

File: 2020/27xxx/CVE-2020-27858.json [hunk 1]
```diff
@@ -1,18 +1,67 @@
 {
```
Comment (cve-team) [inline 2020/27xxx/CVE-2020-27858.json:1]: CNA(s) of user (zdi) do not match owning CNA.

File: 2020/27xxx/CVE-2020-27859.json [hunk 2]
```diff
@@ -1,18 +1,67 @@
 {
```
Comment (cve-team) [inline 2020/27xxx/CVE-2020-27859.json:1]: CNA(s) of user (zdi) do not match owning CNA.

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2020/27xxx/CVE-2020-27858.json (hunk 1 above)
Comment (zdi-team) [reply 2020/27xxx/CVE-2020-27858.json:1]: We don't know how to remedy this. Our username is zdi-team, which we've used before, and we are the "CNA of last resort" for vulnerabilities that come through the ZDI program as listed here: https://cve.mitre.org/cve/request_id.html#cna_participants

**Review by cve-team — APPROVED** [review]

**Pull request merged**